ansible/group_vars/sonic/sku-sensors-data.yml

```diff
@@ -149,6 +149,8 @@ sensors_checks:
 
     psu_skips: {}
 
+    sensor_skip_version: {}
```

Comment:
Maybe we don't need add so many empty "sensor_skip_version" section. How about this: if the key "sensor_skip_version" does not exist, we skip nothing.

Reply:
this will be a reminder to all the vendors that a new function has been added,  same as the previously added "psu_skip" list.